infrastructure/aws/lambda/handler.py

```diff
@@ -4,37 +4,24 @@
 import logging
 import os
 import warnings
-from typing import Any, Dict
+from typing import Any
 
-import earthaccess
 from mangum import Mangum
+from mangum.types import LambdaContext, LambdaEvent
 
 from titiler.cmr.logger import configure_logging
 from titiler.cmr.main import app
-from titiler.cmr.settings import AuthSettings
 
 configure_logging()
 
 warnings.filterwarnings("ignore", category=UserWarning)
 warnings.filterwarnings("ignore", category=FutureWarning)
 logging.getLogger("numexpr").setLevel(logging.WARNING)
 
-auth_config = AuthSettings()
-
-
-@app.on_event("startup")
-async def startup_event() -> None:
-    """startup."""
-    if auth_config.strategy == "environment":
-        app.state.cmr_auth = earthaccess.login(strategy="environment")
-    else:
-        app.state.cmr_auth = None
-
 
 handler = Mangum(
     app,
     lifespan="off",
-    api_gateway_base_path=None,
```

Comment:
This line didn't pass type checking because `api_gateway_base_path` is typed as `str`, not `str | None`. However, removing the line does not change the behavior because whether the argument is passed as `None` (before) or not passed at all (now), it defaults to `"/"`.